user_docs/en/userGuide.t2t

```diff
@@ -2925,6 +2926,30 @@ If you install an add-on with paid components and change your mind about using i
 The Add-on Store is accessed from the Tools submenu of the NVDA menu.
 To access the Add-on Store from anywhere, assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
+++ Note on security and privacy when using Add-ons ++[AddonSecurityAndPrivacy]
+Installing add-ons in NVDA leads to integration of external code into NVDA's functionality in order to enhance NVDA or make new features possible.
+Add-ons can also use external libraries and third party services to serve the purpose and provide the features for which they have been developed. They can be developed by every person or company.
+
+Automated code checks for add-ons happen when they are submitted to NVDA’s Add-on Store. This means e.g.  they will undergo  a basic automated check for vulnerabilities by security analysis tools such as codeQL.
+There isn't currently any formal review process for add-ons performed by a human.
+However, many add-ons have discussions areas where users can exchange feedback. The [community review area #AddonStoreReviews] can be accessed via the actions menu of the add-on.
+
+Installed Add-ons or extensions (not only in NVDA) might in general introduce security and/or privacy vulnerabilities, depending on the permissions they need and actions they perform in order to provide the desired functionality.

```

Comment:
I also find this term "permissions" concerning. That is not a concept that Windows uses in the way that you're using it here, and I think it is just another ill-defined term that we then have to manage concerns around.

Microsoft doesn't define it. NV Access doesn't define it.
And add-on authors haven't been told that they should be aware of it.

Warning the users about something that the add-on authors haven't been asked to carefully document and warn about, is asking for all kinds of trouble.

Those comments aside, I have restated this somewhat as a suggestion if you decide to keep it. Though I would really suggest finding a way to say it without invoking "permissions", which has specific meanings in computing.

```suggestion
As a general rule, NVDA's add-ons or third-party extensions in other software such as browsers, have the potential to introduce security and/or privacy vulnerabilities, depending on the permissions they need and actions they perform in order to provide their stated functionality.
```

Reply:
This statement is not true. Microsoft defines permissions and in general it is also a general knowledge that everyone should have regardless of NVDA. It is true we cannot use a permissions system to enable or disable such things per add-on but still it is a term that is important to keep in mind when reading add-ons descriptions.
https://support.microsoft.com/en-us/windows/app-permissions-aea98a7c-b61a-1930-6ed0-47f0ed2ee15c

Reply:
I strongly disagree with the word of "permission" as it is used in the current version of this PR. 

The link you have provided seems to apply to Windows Store apps, what does not apply to NVDA and its add-ons; they write:
> The Privacy page won't list apps with permission to use all system resources. You can't use the Privacy settings to control what capabilities these apps can use. Windows Desktop apps fall under this category.

Using "permission" with NVDA for which it does not apply is confusing; it's even more confusing since you speak of permission of add-ons which are not even applications/apps. Many people are not aware that there is a permission system in Windows, whereas they are used to it on smartphones. So using "permission" where is these do not apply just make things more confusing.

When I first saw the concept of permission in this PR, things were not clear to me, and I consider myself an advanced user; what about all other users?

Here I'd just remove "permission they need and".

What is true is that all add-ons have the same permissions as NVDA and any other desktop application:
* they can access to the file system (read/write)
* they can use the mic and the camera
* they can launch an application
* they can establish internet connection
* etc.

The only permissions that I can see are the one of the file system (linked to the user) and the one of the UAC.